_data/catalog/widgets.json

```diff
@@ -0,0 +1,1569 @@
+    "categories": [
+      "Stack"
+    ],
+    "subcategories": [
+      "Multi-child layout widgets"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/Stack-class.html",
+    "image": "<svg viewBox='0 0 100 100'><rect x='0' y='0' width='100' height='100' fill='#3949ab'/><rect x='10' y='10' width='80' height='80' fill='#ffffff'/><rect x='25' y='25' width='40' height='30' fill='#3b75ad'/><rect x='45' y='35' width='30' height='30' fill='#4dd0e1'/><rect x='40' y='50' width='10' height='10' fill='#f50057'/></svg>"
+  },
+  {
+    "name": "Placeholder",
+    "description": "A widget that draws a box that represents where other widgets will one day be added.",
+    "categories": [
+      "Basics"
+    ],
+    "subcategories": [],
+    "link": "https://docs.flutter.io/flutter/widgets/Placeholder-class.html",
+    "image": "/images/catalog-widget-placeholder.png"
+  },
+  {
+    "name": "BottomNavigationBar",
+    "description": "Bottom navigation bars make it easy to explore and switch between top-level views in a single tap. The BottomNavigationBar widget implements this component.",
+    "categories": [],
+    "subcategories": [
+      "App structure and navigation"
+    ],
+    "link": "https://docs.flutter.io/flutter/material/BottomNavigationBar-class.html",
+    "image": "https://material-design.storage.googleapis.com/publish/material_v_9/0B7WCemMG6e0VWG5nei0wWXpoczA/components_bottom_navigation.png"
+  },
+  {
+    "name": "TabBar",
+    "description": "A material design widget that displays a horizontal row of tabs.",
+    "categories": [],
+    "subcategories": [
+      "App structure and navigation"
+    ],
+    "link": "https://docs.flutter.io/flutter/material/TabBar-class.html",
+    "image": "https://material-design.storage.googleapis.com/publish/material_v_9/0B7WCemMG6e0VaWdBdnhMT3ViXzQ/components_tabs.png"
+  },
+  {
+    "name": "TabBarView",
+    "description": "A page view that displays the widget which corresponds to the currently selected tab. Typically used in conjuction with a TabBar.",
+    "categories": [],
+    "subcategories": [
+      "App structure and navigation"
+    ],
+    "link": "https://docs.flutter.io/flutter/material/TabBarView-class.html",
+    "image": "https://storage.googleapis.com/material-design/publish/material_v_11/assets/0B7WCemMG6e0VaWdBdnhMT3ViXzQ/components_tabs.png"
+  },
+  {
+    "name": "MaterialApp",
+    "description": "A convenience widget that wraps a number of widgets that are commonly required for material design applications.",
+    "categories": [],
+    "subcategories": [
+      "App structure and navigation"
+    ],
+    "link": "https://docs.flutter.io/flutter/material/MaterialApp-class.html",
+    "image": "https://storage.googleapis.com/material-design/publish/material_v_11/assets/0Bx4BSt6jniD7Y1huOXVQdlFPMmM/materialdesign_introduction.png"
+  },
+  {
+    "name": "WidgetsApp",
```

Comment:
WidgetsApp is not really a Material Design widget. Let's move it to "See also" at the end of this sub-category, once we have the ability to do so.